packages/custom-functions-metadata/src/parseTree.ts

```diff
@@ -431,11 +553,11 @@ function validateId(
     if (!id) {
       id = "Function name is invalid";
     }
-    const errorString = `The custom function id contains invalid characters. Allowed characters are ('A-Z','a-z','0-9','.','_'):${id}`;
+    const errorString = `The custom function or enum id contains invalid characters. Allowed characters are ('A-Z','a-z','0-9','.','_'):${id}`;
```

Comment:
nit: if you can tell whether it is a custom function or an enum, it would be nice that the error string was formatted to just say the one that it is.